.eslintrc.js

```diff
@@ -277,6 +270,67 @@ module.exports = {
         'import/no-default-export': 'off',
       },
     },
+    {
+      // TODO: fix the issues in these files
```

Comment:
Using the same strategy as with TypeScript strict migration. By default: enforcing the rule. But still allowing it to be broken in non-migrated files that we can slowly port over instead of doing one-big-bang.